src/redux/reducers/auth.reducer.js

```diff
@@ -4,6 +4,8 @@ import {
   SIGNUP_SUCCESS,
   VERIFIED,
   VERIFICATION_FAILED,
+  LOG_OUT_SUCCESS,
+  LOG_OUT,
```

Comment:
@EmyRukundo, Could you please help me understand the difference between these two action types?

Reply:
@eliemugenzi  Log_out_success happen when the user logout successful while log_out happens when user tried to logout but fails